src/http/controllers/user/nurseUser.js

```diff
@@ -25,6 +25,29 @@ const createNurseUser = async (req, res) => {
   }
 };
 
+/**
+ * @description Controller for "get all nurse user" API operation
+ * @param {object} req Express request object
+ * @param {object} res Express response object
+ */
+const getAllNurseUser = async (req, res) => {
+  const { log, user: { _id: wardId, hospitalId } } = res.locals;
```

Comment:
If it's an hospitalUser that initiated this request, the `_id` field here would be the hospitalId and not wardId, I don't understand why you just assumed the `_id` field would always be the wardId